### DISCUSSION TITLE
Help creating a rule for saving images from Bluesky

### DISCUSSION THREAD

**Discussion by seth-rah** [gyng/save-in#219] [Q&A] [answered]
So there are some nice templates available here

https://github.com/gyng/save-in/wiki/Filename-rewrite

But it's not working right now for media saved from Bluesky.

File extensions seem to be stored as an @ instead of with a fullstop, any help appreciated.

**Comment by rudolphos** [comment]
This works when you are on the post page, for example `https://bsky.app/profile/bsky.app/post/3l6dplwluhb2f`

```
// Bluesky post
filename: .*
pagedomain: bsky.app/profile/(.*)
into: :pagetitle:.jpg
```

Saved filename looks like this:

`Bluesky_ _📢 App Version 1.92.1 is rolling out now (1_2) This release includes multiple bugfixes and UI tweaks.jpg`

**Reply by seth-rah** [reply, marked as answer]
It's been a while 

But I have this configured right now

```
// Bluesky post
// https://cdn.bsky.app/img/feed_thumbnail/plain/did:plc:y354byzmbnusaeqqmwuftztd/bafkreiap75eecj3iywani6vm5dwz3bnrnwxjimkqbpu4ok3ddzfai3t5gq@jpeg
// into 20250318T230501Z_bafkreiap75eecj3iywani6vm5dwz3bnrnwxjimkqbpu4ok3ddzfai3t5gq.jpeg
sourceurl: cdn.bsky.app/.*/([^/@]+)@([^/@]+)
capture: sourceurl
into: :isodate:_:$1:.:$2:
```
